src/vue/pages/VerificationFormContainer.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="verification-form-container">
+    <h5 class="verification-form-container__heading">
+      {{ 'general-form.account-information-lbl' | globalize }}
+    </h5>
+
+    <template v-if="formToShow === 'general'">
+      <kyc-general-form
+        class="verification-form-container__form"
+        :former="createGeneralFormer()"
+        @submitted="onFormSubmit"
+      />
+    </template>
+
+    <template v-else-if="formToShow === 'corporate'">
+      <kyc-corporate-form
+        class="verification-form-container__form"
+        :former="createCorporateFormer()"
+        @submitted="onFormSubmit"
+      />
+    </template>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import { vuexTypes } from '@/vuex'
+import { scrollToTop } from '@/js/helpers/scroll-helpers'
+import { KycGeneralFormer } from '@/js/formers/KycGeneralFormer'
+import { KycCorporateFormer } from '@/js/formers/KycCorporateFormer'
+import { vueRoutes } from '@/vue-router/routes'
+import KycGeneralForm from '@/vue/forms/KycGeneralForm'
+import KycCorporateForm from '@/vue/forms/KycCorporateForm'
+
+export default {
+  name: 'verification-form-container',
+
+  components: {
+    KycGeneralForm,
+    KycCorporateForm,
+  },
+
+  computed: {
+    ...mapGetters({ kycRequest: vuexTypes.kycRequest }),
+
+    formToShow () {
+      switch (this.$route.name) {
```

Comment:
I vote for having 2 separate route components instead of such strange if's.

Reply:
done